prov/verbs/src/fi_verbs.c

```diff
@@ -64,6 +64,13 @@ struct fi_ibv_gl_data fi_ibv_gl_data = {
 		.use_name_server	= 1,
 		.name_server_port	= 5678,
 	},
+
+	.msg			= {
+		/* Disabled by default. Use XRC transport for message
+		 * endpoint only if it is explicitly requested */
+		.use_xrc		= 0,
+		.xrcd_filename		= "/tmp/verbs_xrcd",
```

Comment:
Yes they can if they have adequate file access permissions, or they can be instructed to use different files through the setting of FI_VERBS_EXRD_FILENAME environment variable.  In theory, if the jobs can run as separate users then you could use different filenames to further partition the work; in practice this shouldn't be necessary since the Verbs provider is controlling the TGT QP to be shared.